### PR TITLE
enh(core): add mechanism to get frontend translation of extensions

### PR DESCRIPTION
**Pull request by kduret**
<h2> Description </h2>

* Get translation of extensions
* Remove expose-loader and create directly "external" chunk in source code
* Minimize bundle size by using timezone set of 10 years

<h2> Type of change </h2>

- [ ] Patch fixing an issue (non-breaking change)
- [x] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

<h2> Target serie </h2>

- [ ] 2.8.x
- [ ] 18.10.x
- [x] 19.04.x (master)

<h2> How this pull request can be tested ? </h2>

Try an extension with frontend translation (eg: autodiscovery)

### CODE REVIEW

File: src/Centreon/Application/Webservice/CentreonI18n.php [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?php
 /**
- * Copyright 2005-2018 Centreon
+ * Copyright 2005-2019 Centreon
  * Centreon is developped by : Julien Mathis and Romain Le Merlus under
```
Comment (sc979) [inline src/Centreon/Application/Webservice/CentreonI18n.php]:
```suggestion
 * Centreon is developed by : Julien Mathis and Romain Le Merlus under
```

File: src/Centreon/Tests/Domain/Service/I18nTest.php [hunk 2]
```diff
@@ -0,0 +1,95 @@
+<?php
+/*
+ * Copyright 2005-2019 Centreon
+ * Centreon is developped by : Julien Mathis and Romain Le Merlus under
```
Comment (sc979) [inline src/Centreon/Tests/Domain/Service/I18nTest.php]:
```suggestion
 * Centreon is developed by : Julien Mathis and Romain Le Merlus under
```

File: www/front_src/src/helpers/dynamicImport.js [hunk 3]
```diff
@@ -17,18 +19,22 @@ export function dynamicImport(basename, parameters) {
...
+        if (module.default && typeof(module.default) === 'object') { // named umd export
+          window[vector] = module.default;
+        } else { // unnamed umd export or systemjs export
+          window[vector] = module;
+        }
         return resolve(window[vector]);
       }
     } catch (error) {
-      return reject(error);
+      console.error(error);
```
Comment (sc979) [inline www/front_src/src/helpers/dynamicImport.js]: please be more specific, ie:  "dynamicImport error : " + error

Comment (kduret) [reply www/front_src/src/helpers/dynamicImport.js]: error is an object, not a string.
I will print the stack trace, so no need to display more information

**Pull request merged**